### PR TITLE
1st edit index file

### PR DESCRIPTION
**Pull request by mary-tziraki**
I update index with the info we have so far. 
I haven't added eventbrite registration number. 
I added as online store the 'faculty of life sciences ' as before. We might need to change it

<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

If this contribution is for instructor training, please email the link to this contribution to
instructor.training@carpentries.org so we can record your progress. You've completed your contribution
step for instructor checkout by submitting this contribution!

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
respond to your contribution. Although not all contributions can be incorporated into the lesson
materials, we appreciate your time and effort to improve the curriculum. If you have any questions
about the lesson maintenance process or would like to volunteer your time as a contribution
reviewer, please contact The Carpentries Team at team@carpentries.org.

You may delete these instructions from your comment.

\- The Carpentries
</details>

### CODE REVIEW

**Comment by mary-tziraki** [comment]
I have updated the /_includes/swc/schedule.html 

Could you please check if it is right or needs more things to add/remove? 
Thank you

File: index.md [hunk 1]
```diff
@@ -94,7 +95,7 @@ displayed if the 'eventbrite' field in the header is not set.
 <strong>Some adblockers block the registration window. If you do not see the
   registration box below, please check your adblocker settings.</strong>
 <iframe
-  src="https://www.eventbrite.com/tickets-external?eid={{page.eventbrite}}&ref=etckt"
+  src="https://www.eventbrite.co.uk/e/ucl-dash-workshop-in-python-health-care-data-analysis-and-medical-imaging-tickets-539618031057"
```
Comment (dpshelio) [inline index.md:77]: The idea is that you leave this as it was: 
```suggestion
  src="https://www.eventbrite.com/tickets-external?eid={{ page.eventbrite }}&ref=etckt
```
and add the code  to the line 20.

File: index.md [hunk 2]
```diff
@@ -3,21 +3,22 @@ layout: workshop      # DON'T CHANGE THIS.
...
+startdate: 2023-03-09      # machine-readable start date for the workshop in YYYY-MM-DD format like 2015-01-01
+enddate: 2023-03-14         # machine-readable end date for the workshop in YYYY-MM-DD format like 2015-01-02
+instructor: ["David Pérez-Suárez","Mary Tziraki", "Thomas Veale", "Gerold Baier", "Saba Ferdous", "Adam Lee"] # boxed, comma-separated list of instructors' names as strings, like ["Kay McNulty", "Betty Jennings", "Betty Snyder"]
+helper: ["TBC"]     # boxed, comma-separated list of helpers' names, like ["Marlyn Wescoff", "Fran Bilas", "Ruth Lichterman"]
+email: ["arc-teaching+carpentries@ucl.ac.uk","m.tziraki@ucl.ac.uk"]    # boxed, comma-separated list of contact email addresses for the host, lead instructor, or whoever else is handling questions, like ["marlyn.wescoff@example.org", "fran.bilas@example.org", "ruth.lichterman@example.org"]
+collaborative_notes:   # optional: URL for the workshop collaborative notes, e.g. an Etherpad or Google Docs document (e.g., https://pad.carpentries.org/2015-01-01-euphoria)
 eventbrite:           # optional: alphanumeric key for Eventbrite registration, e.g., "1234567890AB" (if Eventbrite is being used)
```
Comment (dpshelio) [inline index.md:20]: Adds the codimd and the eventbrite id.
```suggestion
collaborative_notes:  https://codimd.carpentries.org/2023-03-09-UCL-DaSH # optional: URL for the workshop collaborative notes, e.g. an Etherpad or Google Docs document (e.g., https://pad.carpentries.org/2015-01-01-euphoria)
eventbrite:   539618031057        # optional: alphanumeric key for Eventbrite registration, e.g., "1234567890AB" (if Eventbrite is being used)
```

File: index.md [hunk 3]
```diff
@@ -103,6 +104,18 @@ displayed if the 'eventbrite' field in the header is not set.
...
+  <a href="https://onlinestore.ucl.ac.uk/product-catalogue/{{page.onlinestore}}">
+    <button type="button" class="btn btn-info">Register </button>
+  </a>
+</p>
+<p class="text-center">
+  <b>Note: There is no registration fees. Please bring your own refreshments and snacks. </b>
+</p>
+{% endif %}
+
+
```
Comment (dpshelio) [inline index.md]: We don't need this as we are not using ucl's online store. You may want to add the note somewhere in the index page.

**Review by dpshelio — CHANGES_REQUESTED** [review]
Some small changes. I'll add the circle to be able to get the previews working.

**Pull request merged**